fhir2dataset/query.py

```diff
@@ -289,22 +277,22 @@ def _join_2_df(
             how = "inner"
 
         if alias_1 == alias_parent:
-            ### to delete after ??
+            ### to delete after ??  # noqa
             if alias_1 == "patient":
                 df_2 = self._group_lines(df_2, child_on)
             elif alias_2 == "patient":
                 df_1 = self._group_lines(df_1, parent_on)
-            ###################################################
+            ##################  # noqa
```

Comment:
???

Reply:
Indeed it is not beautiful at all. I really wanted to highlight a section of the code that is specific to fhir2ecrf and that we absolutely must not forget to remove in future releases. Do you want me to just delete those comments? Or do you have another idea?

Reply:
I think you can find a way to highlight this part without upsetting the linter
maybe
```
# THIS IS A REALLY IMPORTANT PART BECAUSE .... !!!!!!!
```

Reply:
I really like the idea! It's changed.